superset-frontend/src/components/AsyncSelect/index.jsx

```diff
@@ -21,7 +21,7 @@ import PropTypes from 'prop-types';
 // TODO: refactor this with `import { AsyncSelect } from src/components/Select`
 import { Select } from 'src/components/Select';
 import { t, SupersetClient } from '@superset-ui/core';
-import { getClientErrorObject } from '../utils/getClientErrorObject';
+import { getClientErrorObject } from '../../utils/getClientErrorObject';
```

Comment:
nit: think we want absolute paths here src/utils/getClientErrorObject

Reply:
tagging this with `revisit:src path` for someone else to come along and touch up later. On with the show!